src/ground_truth_nsrts.py

```diff
@@ -1906,6 +1908,59 @@ def _get_repeated_nextto_gt_nsrts() -> Set[NSRT]:
     return nsrts
 
 
+def _get_repeated_nextto_single_option_gt_nsrts() -> Set[NSRT]:
+    """Create ground truth NSRTs for RepeatedNextToSingleOptionEnv."""
+    robot_type, dot_type = _get_types_by_names(CFG.env, ["robot", "dot"])
+
+    NextTo, NextToNothing, Grasped = _get_predicates_by_names(
+        CFG.env, ["NextTo", "NextToNothing", "Grasped"])
+
+    MoveGrasp, = _get_options_by_names(CFG.env, ["MoveGrasp"])
+
+    nsrts = set()
+
+    # Move
```

Comment:
the only things that change are the option and the sampler, right? can we just call the repeatd_nextto_get_gt_nsrts() to get its two NSRTs, and then create new NSRTs using the NSRT constructor, with the option and sampler being the only differences?